lib/src/json_schema/validator.dart

```diff
@@ -95,6 +101,9 @@ class Validator {
 
   bool _treatWarningsAsErrors;
 
+  // Keep track of the number of evaluated items in a list.
+  List<int> _evaluatedItemsContext = [];
```

Comment:
Might be worth explaining the meaning of the indices of the list as part of the comment?

Reply:
I don't think it's worth a dependency is this case.